tensorflow/python/ops/nn_ops.py

```diff
@@ -1836,7 +1836,7 @@ def softmax_cross_entropy_with_logits_v2(
     name: A name for the operation (optional).
 
   Returns:
-    A `Tensor` of the same shape as `labels` and of the same type as `logits`
+    A `Tensor` of shape `[batch_size]` and of the same type as `logits`
```

Comment:
This is only true if labels has rank 2 (although this statement is certainly more accurate than before this PR). If labels has a a higher rank than 2, e.g. if labels is of shape [d0, d1, ..., dn, num_classes], the output will have shape [d0, d1, ..., dn]. Can you clarify this?